rviz_default_plugins/src/rviz_default_plugins/displays/map/map_display.cpp

```diff
@@ -86,6 +86,7 @@ MapDisplay::MapDisplay()
   color_scheme_property_->addOption("map", 0);
   color_scheme_property_->addOption("costmap", 1);
   color_scheme_property_->addOption("raw", 2);
+  color_scheme_property_->addOption("probability", 2);
```

Comment:
I had to change this value to get the feature to work.
```suggestion
  color_scheme_property_->addOption("probability", 3);
```

Reply:
This is true. @rotu can you update this?